docs/user_guide/SIMP_Administration/General_Administration.rst

```diff
@@ -0,0 +1,292 @@
+system and how to manage them. For information on initial certificate setup,
+refer to the :ref:`Certificates` section of :ref:`Client_Management`.
+
+
+Infrastructure Certificates
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Server certificates are the standard :term:`PKI` certificates assigned either
+by an official :term:`CA` (preferred) or generated using the FakeCA utility
+offered by SIMP. Generated certificates are placed in the ``/etc/pki/simp``
+directory of all managed systems.  These certificates are set to expire
+annually. To change this, edit the following files with the number of days for
+the desired lifespan of the certificates:
+
+.. NOTE::
+
+   This assumes that the user has generated Certificates with the FakeCA
+   provided by SIMP. If official certificates are being used, these settings
+   **must be changed within the official CA, not on the SIMP system**.
+
+-  ``/var/simp/environments/simp/FakeCA/CA``
+
+-  ``/var/simp/environments/simp/FakeCA/ca.cnf``
+
+-  ``/var/simp/environments/simp/FakeCA/default\_altnames.cnf``
+
+-  ``/var/simp/environments/simp/FakeCA/default.cnf``
+
+-  ``/var/simp/environments/simp/FakeCA/user.cnf``
+
+In addition, any certificates that have already been created and signed will
+have a config file containing all of its details in
+``/var/simp/environments/simp/FakeCA/output/conf/``.
+
+.. IMPORTANT::
+
+   Editing any entries in the above mentioned config files will **not** affect
+   existing certificates. Existing certificates must be regenerated if you need
+   to make changes.
+
+The following is an example of how to change the expiration time from one year
+(the default) to five years for any newly created certificate.
+
+.. code-block:: bash
+
+  for file in $(grep -rl 365 /var/simp/environments/simp/FakeCA/)
+  do
+    sed -i 's/365/1825/' $file
+  done
+
+
+Puppet Certificates
+^^^^^^^^^^^^^^^^^^^
+
+Puppet certificates are issued and maintained strictly within Puppet.  They are
+different from the server certificates and should be managed with the
+``puppet cert`` utility.
+
+For documentation on the ``puppet cert`` tool, visit the `Puppet Inc. cert manual`_.
+
+You can find the location for the Puppet certificates on your system by running
+``puppet config print ssldir``.
+
+.. NOTE::
+
+   By default, Puppet certificates expire every five (5) years.
+
+
+The SIMP Utility
+----------------
+
+The SIMP server provides a command line utility called ``simp`` that is an
+interface into SIMP-specific settings and subsystems.
+
+You can get information on the ``simp`` utility by running ``simp help`` on
+your SIMP server.
+
+.. _simp passgen:
+
+simp passgen
+^^^^^^^^^^^^
+
+Throughout the SIMP codebase, you may find references to the ``passgen()``
+function. This function auto-generates passwords and stores them in
+``/var/simp/environments/<client environment>/simp_autofiles/gen_passwd``
+on the Puppet server.
+
+For more information, see the `passgen()`_ documentation.
+
```

Comment:
Do you want to list all the commands provided here?
- simp config -> refer to documentation in other sections?
- simp bootstrap 
- simp doc (don't know if it works)